centreon/tests/e2e/cypress/e2e/Resources-status/04-timezone/index.ts

```diff
@@ -0,0 +1,406 @@
+import { Given, Then, When } from '@badeball/cypress-cucumber-preprocessor';
+
+import {
+  checkServicesAreMonitored,
+  submitResultsViaClapi,
+  updateFixturesResult
+} from '../../../commons';
+import {
+  actionBackgroundColors,
+  insertDtResources,
+  secondServiceInDtName,
+  serviceInAcknowledgementName,
+  serviceInDtName,
+  tearDownResource
+} from '../common';
+
+const chosenTZ = 'Africa/Casablanca';
+
+const roundDateTime = (date: Date): string => {
+  return new Date(
+    date.getFullYear(),
+    date.getMonth(),
+    date.getDate(),
+    date.getHours(),
+    Math.ceil(date.getMinutes())
+  ).toLocaleTimeString('en-US', {
+    hour: 'numeric',
+    minute: '2-digit'
+  });
+};
+
+const covert12hFormatToDate = (timeString: string): Date => {
+  const dateString = 'June 8, 2023';
```

Comment:
we are the 9th of June, does it still work ? xD

Reply:
@kduret the idea here was to take the time from the header which is a 12h Format time and convert it to a date but to be able to do this, the string needs to be completed (with the year and date ect) and yes it works haha